body.tex

```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
 agile way, while also ensuring that scope, cost and schedule are
 carefully controlled.
 
+
+\subsection{Defining the Schedule with Milestones}
+
+Rubin Milestones are defined as Jira issues of type ``Milestone''. As indicated above, the Director (or their designate) defines the L1 milestones, the Associate Directors (ADs) define their departments' L2 milestones, and the Team Leaders and ADs define the L3 milestones for their teams.
```

Comment:
We do the initial Milestone defining in smartsheet (this is the tool we will use to do FY milestone planning.  Once all the milestones are input and ADs and team leads have coordinated and the Director approves, then Cathy creates the Jira Milestone Issues using the info in the smartsheet.

Reply:
I added a note on this - the instructions for milestone issue creation are still needed I think, if only for a future Program Coordinator!